### ISSUE TITLE
Program will not launch

### ISSUE THREAD

**Issue by preludeinz** [writemonkey/wm3#319]
Computer crashed, and now the program and previous versions will not launch reliably. When the program does open, it doesn't go past the monkeywisdom screen and can't seem to find my last session. I've lost about twenty documents and don't know how to recover them without being able to open the program.

**Comment by writemonkey** [comment]
Did computer crashed when wm was active?
i.

**Comment by preludeinz** [comment]
Yes.

**Comment by writemonkey** [comment]
Do you remember what exactly were you doing  in wm when it happened?

To restore: go to backup directory and store all files somewhere safe. Then replace the database files with a copy of backup files. Restart wm.
i.

**Comment by preludeinz** [comment]
I was just using writemonkey as usual, I don't think writemonkey was the program that caused the computer to crash. By backup directory, I assume you mean the files in _WRITEMONKEY3_BACKUP in the user directory?

**Comment by writemonkey** [comment]
Yes, but save existing database files before replacing them with backup ones. Also try files with _OLDER in name (remove _older part).
i.

**Comment by preludeinz** [comment]
So far no luck, the program still won't launch. I've mostly resolved myself to losing the database files, even though it's a shame.

**Comment by preludeinz** [comment]
I've managed to get the program to launch, but it seems like it's corrupted the file it's trying to open to, and I can't interact with the UI at all. Pixelation is mine, everything else is the way it looks when it launches. it was also only an older build of wm3 that I was able to launch, January 2019, iirc.

![image](https://user-images.githubusercontent.com/26475621/65367053-e5c8ea00-dbf9-11e9-8eec-73c10658d294.png)
